app/Views/errors/cli/error_exception.php

```diff
@@ -50,7 +50,7 @@
             $function .= $padClass . $error['function'];
         }
 
-        $args = implode(', ', array_map(static fn ($value) => match (true) {
+        $args = implode(', ', array_map(static fn ($value): ?string => match (true) {
```

Comment:
How can this return `null`?

Reply:
`var_export()` on default fallback return `?string` , see https://www.php.net/manual/en/function.var-export.php

Reply:
I think it's a conditional return. when `$return` is `true` it will return the string representation of the variable. otherwise it prints it and returns nothing (so `null`)

Reply:
ok, updated https://github.com/codeigniter4/CodeIgniter4/pull/9343/commits/47d69b6080d565847e8e1c0353934fee31a55615